api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -257,6 +257,20 @@ public OfflineUserController(
     return ResponseEntity.noContent().build();
   }
 
+  @Override
+  public ResponseEntity<Void> expireNonCompliantUsers() {
```

Comment:
Please add a comment explaining why this will expire users - it's not obvious if you're not familiar with how  `updateUserWithRetries()` works.

Reply:
Thinking ahead (not for this PR), could we rename this function so one does not have to add a comment whenever one uses it? Perhaps split into two: `applyToUser` (pass function) and `applyDefaultUpdatesToUser`? I'm omitting "withRetries" since I'm not sure it makes sense to expose that implementation detail.

Reply:
Yeah - this method and related methods could use a rename/rethink.

But in many cases the current name works: it updates the user with the given update.  As a side effect, it also checks and enforces the user's access tier status.

Here we're using it *for the sole purpose of its side effect* so "update user" is too vague.

Reply:
Added comment